backstop/tests/header.html

```diff
@@ -966,17 +967,24 @@ <h2>Split Menu Buttons</h2>
 
           <!-- Project Button -->
           <nav aria-label="breadcrumbs">
-            <button
-              class="iui-button iui-borderless iui-header-button iui-large"
-              type="button"
-              aria-label="Project picker"
-              aria-expanded="false"
-            >
-              <span class="iui-label">
-                <div class="iui-name">Project Alpha</div>
-              </span>
+            <span class="iui-button-split-menu">
+              <button
+                class="iui-button  iui-borderless iui-header-button iui-large"
+                type="button"
+                aria-label="Project picker"
+                aria-expanded="false"
+                id="test-button-5"
+              >
+                <span class="iui-label">
+                  <div>Project Alpha</div>
+                </span>
+              </button>
               <!-- Use caret-down-small.svg from @itwin/itwinui-icons package. -->
```

Comment:
Can you move these comments down right where the svg is actually used? Saw it in a couple more places (like line 799).

Like this:

```html
<!-- Use caret-down-small.svg ...>
<svg ...>
```